grisera_api/participation/participation_service_graphdb.py

```diff
@@ -18,8 +18,11 @@ class ParticipationServiceGraphDB(ParticipationService):
     participant_state_service (ParticipantStateService): Service to send participant state requests
     """
     graph_api_service = GraphApiService()
-    activity_execution_service = ActivityExecutionServiceGraphDB()
-    participant_state_service = ParticipantStateServiceGraphDB()
+
+    def __init__(self):
+        self.activity_execution_service = None
+        self.participant_state_service = None
+        self.recording_service = None
```

Comment:
Missing type definition of these services

Reply:
As above